src/pages/installation.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: ~/layouts/Main.astro
+---
+
+# Installing Astro
+
+Welcome to Astro, we are really excited that you are here and would like to assist you in getting up and started with Astro in no time.
+
+<!--
+    Quick Install
+    Manual Install
+    Troubleshooting
+ -->
+
+## Preflight
+
+Before we begin, let us check to make sure that everything is in order.
+
+To use Astro you need to have Node.js already installed and working on your machine. If you haven't got Node.js installed, you can download it from their site.
```

Comment:
I think a link for the node page is important here.

Reply:
I had forgot to add that one in, 😅
Its in with the new one......promise,